theme/styles.ts

```diff
@@ -1,1043 +1,1030 @@
-import { background, flexbox } from "@chakra-ui/react";
+/*
+ TODO: TURN MOST OF THESE STYLES INTO NON-GLOBAL STYLES BY USING COMPONENTS OR STYLE MIXINS
 
+ PLEASE TRY NOT TO ADD ANYTHING TO THIS FILE.
+*/
```

Comment:
For the remainder of the season, I will be going maintenance mode

- Baking this file into chakra styles is one of the main items I will address.